docs/user/commandline_tools.rst

```diff
@@ -420,6 +422,114 @@ Explain how a given GeoMesa query will be executed::
       -c test_catalog -f test_feature \
       -q "INTERSECTS(geom, POLYGON ((41 28, 42 28, 42 29, 41 29, 41 28)))"
 
+stats-analyze
+~~~~~~~~~~~~~
+
+Analyze statistics for your data set. This may improve query planning.
+
+Example usage::
+
+    $ geomesa stats-analyze -u username -p password -c geomesa.data -f twitter
+      Running stat analysis for feature type twitter...
+      Stats analyzed:
+        Total features: 8852601
+        Bounds for geom: [ -171.75, -45.5903996, 157.7302, 89.99997102 ]
+        Bounds for dtg: [ '2016-02-01T00:09:12.000Z' to '2016-03-01T00:21:02.000Z' ]
+        Bounds for user_id: [ '100000215' to '99999502' ]
+      Use 'stats-histogram' or 'stats-count' commands for more details
+
+stats-bounds
+~~~~~~~~~~~~
+
+Displays the bounds of your data for different attributes. You can use pre-calculated stats for a quick
+estimation, or get the definitive result by querying the data set using the '--exact' flag.
+
+Example usage::
+
+    $ geomesa stats-bounds -u username -p password -i instance -z zoo1,zoo2,zoo3 \
+        -c geomesa.data -f twitter
+      user_id [ 100000215 to 99999502 ] cardinality: 861283
+      user_name [ unavailable ]
+      text [ unavailable ]
+      dtg [ 2016-02-01T00:09:12.000Z to 2016-03-01T00:21:02.000Z ] cardinality: 2161132
+      geom [ -171.75, -45.5903996, 157.7302, 89.99997102 ] cardinality: 2119237
+
+    $ geomesa stats-bounds -u username -p password -i instance -z zoo1,zoo2,zoo3 \
+        -c geomesa.data -f twitter --exact \
+        -q 'BBOX(geom,-70,45,-60,55) AND dtg DURING 2016-02-02T00:00:00.000Z/2016-02-03T00:00:00.000Z'
+      Running stat query...
+        user_id [ 1011811424 to 99124417 ] cardinality: 115
+        user_name [ foo_user to bar_user ] cardinality: 113
```

Comment:
fixed here and the line below and the same lines in the tools readme - let me know if you spot any other places.